index.js

```diff
@@ -60,7 +64,11 @@ function config(opts) {
             cfg[name] = value;
         }
         else if ( opt.type === 'boolean' ) {
-            booleanValue = valid.boolean[value.toLowerCase()];
+            if ( typeOf(value) != 'boolean' ) {
```

Comment:
OFF topic, is there a chance to specify "multi-level" configs?
Single level in my mind:
{ key: value,
  key: value
}

Multi-level:
{ aws: {
             cred: {
                       access_key: ENV VARIABLE,
                       secret_key: ENV VARIABLE,
                       region: ENV VARIABLE,
             }
     something_else: {
 }
}